tests/ecalc_cli/snapshots/test_app/test_new_stp_export_properly/test.INST_A.stp.tsv

```diff
@@ -145,14 +145,14 @@ forecastYear	forecastMonth	co2Emission	methaneEmission
 2032	11	34854.68	27.95728
 2032	12	36016.5	28.88918
 2033	01	5219.94	2.251254
-2033	02	0	0
-2033	03	0	0
-2033	04	0	0
-2033	05	0	0
-2033	06	0	0
-2033	07	0	0
-2033	08	0	0
-2033	09	0	0
-2033	10	0	0
-2033	11	0	0
-2033	12	0	0
+2033	02	4714.785	2.033391
```

Comment:
Here eCalc is run with MONTH as input frequency, EXTRAPOLATION is FALSE for time series, and END is specified to be 2034-01-01. This means that the input from 2033-01-01 will give values for all of 2033. Earlier the first day of each month would be added to the global time vector, so the values for the last date in the input would only be valid for Jan 2033 - the rest of the months would be set to 0.